SIS_import/sis_set_url.rb

```diff
@@ -11,6 +12,54 @@
 require "uri"
 require "time"
 
+# variables for ESB API call throttling
```

Comment:
This and canvas throttle are nearly identical.  Make a separate class for clarity and to avoid typos and extra maintenance..